### ISSUE TITLE
output of `dokku nginx` and `dokku ssh-keys`

### ISSUE THREAD

**Issue by znz** [dokku/dokku#2627]
Description of problem:
Default of some commands display single line help without `column -c2 -t -s,`.
Is this intentional?

```
vagrant@dokku:~$ dokku nginx
    nginx, Interact with Dokku's Nginx proxy
vagrant@dokku:~$ dokku ssh-keys
    ssh-keys, Manage public ssh keys that are allowed to connect to Dokku
```

How was Dokku installed?: `vagrant up`

**Comment by josegonzalez** [comment]
Can you include *all* of the information we ask for in our [issue template](https://github.com/dokku/dokku/blob/master/ISSUE_TEMPLATE.md)? It is helpful for debugging problems. Thanks.

**Comment by znz** [comment]
output of `dokku report`: https://gist.github.com/znz/a1ad0a86e5a3723741a528f76660e32a

Environment details (AWS, VirtualBox, physical, etc.): VirtualBox

How was Dokku installed?: `vagrant up`

How reproducible:


Steps to Reproduce:
1. `vagrant up`
2. `vagrant ssh`
3. `dokku nginx` and/or `dokku ssh-keys`

Actual Results:
```
vagrant@dokku:~$ dokku nginx
    nginx, Interact with Dokku's Nginx proxy
vagrant@dokku:~$ dokku ssh-keys
    ssh-keys, Manage public ssh keys that are allowed to connect to Dokku
```

Expected Results:

```
vagrant@dokku:~$ dokku ssh-keys | column -c2 -t -s,
    ssh-keys   Manage public ssh keys that are allowed to connect to Dokku
```

or something else.

**Comment by josegonzalez** [comment]
Closed by #2650.
